Docker/README.md

```diff
@@ -1,75 +1,19 @@
 # Docker deployment instructions
```

Comment:
We should probably reflect the changes in the main README.md as well. Or at least leave a reference to the readme that currently lives in the docker directory.

Wiki entry might be appropriate as well.

Reply:
Alright i did the main README.md for now. Just wanted to post that here.